src/params.rs

```diff
@@ -5,11 +5,15 @@ pub type Num = i32;
 pub struct Input {
     pub name: &'static str,
     pub vec: &'static [Num],
+    pub min_uses: u8,
+    pub max_uses: u8,
```

Comment:
could you add some checks to `validate_input()`, for example `max_uses` can't be 0, total `min_uses` can't be too large compared to `MAX_LENGTH`